lib/representable/logger.rb

```diff
@@ -0,0 +1,88 @@
+# Provides really basic logging functionality.
```

Comment:
Need to think more about whether this is the right approach, but 💯 for the great documentation!